src/sysdolphin/baselib/jobj.c

```diff
@@ -170,18 +175,12 @@ void HSD_JObjMakeMatrix(HSD_JObj* jobj)
 
 void HSD_JObjRemoveAnimByFlags(HSD_JObj* jobj, u32 flags)
 {
-    BOOL has_dobj;
     if (jobj != NULL) {
         if (flags & 1) {
             HSD_AObjRemove(jobj->aobj);
             jobj->aobj = NULL;
         }
-        if (jobj->flags & 0x4020) {
-            has_dobj = FALSE;
-        } else {
-            has_dobj = TRUE;
-        }
-        if (has_dobj) {
+        if (jobj->flags & 0x4020 ? FALSE : TRUE) {
```

Comment:
I see this a lot everywhere so it should be a macro. Not sure if the `x & flags ? FALSE : TRUE` can be the same macro as `x & flags ? TRUE : FALSE` though.

Reply:
It's probably the actual definition of `union_type_dobj`. Lots of other macros need to be changed based on assert strings, was going to make another pass to clean these up.